lib/fastlane/plugin/appcenter/actions/appcenter_codepush_release_react.rb

```diff
@@ -0,0 +1,156 @@
+require 'fastlane_core/ui/ui'
+
+module Fastlane
+  UI = FastlaneCore::UI unless Fastlane.const_defined?("UI")
+
+  module Actions
+    class AppcenterCodepushReleaseReactAction < Action
+      def self.description
+        "CodePush release react action"
+      end
+
+      def self.authors
+        ["Ivan Sokolovskii"]
+      end
+
+      def self.run(params)
+        app = params[:app_name]
```

Comment:
What about change execution directory to parents directory to execute appcenter cli directly?

```suggestion
        Dir.chdir("../".to_s) do
        app = params[:app_name]
```

Reply:
@iv0rish Thank you for your review! What is motivation for changing directory here?

Reply:
Sorry for late response @ivan3177 . Fastlane can't find appcenter cli when I run release-react, so I changed directory where appcenter cli can be executed. You can delete this if it's my own problem.